### ISSUE TITLE
wallet in use error during publish?

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry#346]
tried publishing something else, got a popup about "get_new_address" error - this is in the log. Just had transferred some LBC into the wallet. 
2016-12-16 01:07:21,651 ERROR    lbrynet.lbrynet_daemon.auth.server:164: API Failure: [Error 32] The process cannot access the file because it is being used by another process
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\twisted\python\threadpool.py", line 246, in inContext
  File "C:\Python27\lib\site-packages\twisted\python\threadpool.py", line 262, in <lambda>
  File "C:\Python27\lib\site-packages\twisted\python\context.py", line 118, in callWithContext
  File "C:\Python27\lib\site-packages\twisted\python\context.py", line 81, in callWithContext
  File "C:\Python27\lib\site-packages\lbryum\wallet.py", line 141, in write
WindowsError: [Error 32] The process cannot access the file because it is being used by another process

**Comment by tzarebczan** [comment]
I then retried it, and published something to lbry://five...it says published, took me to My Published screen, but had the name of the file that's already on five, and not my file.  The file shown was Bread (and art film) - with 0% downloaded. 

Hmm.

Here it should have told me about the naming claiming system.  Or it should have mentioned it when I tried to outbid (may not know it's not mine yet?).  But to say that it published, that's odd.  And then show the wrong file, even worse.
